### PR TITLE
Coordinates

### PR DESCRIPTION
**Pull request by maxnoe**
Implements a `CameraCoordinate` astropy.coordinates frame, to be able to convert from astropy sky coordinates to FACT camera coordinates and back.

Add two utility functions to convert columns of dataframes or numpy arrays.

### CODE REVIEW

File: fact/coordinates/representation.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+from astropy.coordinates import BaseRepresentation, CartesianRepresentation
+import astropy.units as u
+from collections import OrderedDict
+from astropy.utils.compat.numpy import broadcast_arrays
+
+
+class PlanarRepresentation(BaseRepresentation):
```
Comment (kbruegge) [inline fact/coordinates/representation.py:7]: I don't understand why this is needed

Comment (maxnoe) [reply fact/coordinates/representation.py:7]: Coordinates need to have a representation, astropy does not provide a 2d Cartesian representation, only 3d. This is basically copy/paste from ctapipe (with a fixed bug).

Comment (kbruegge) [reply fact/coordinates/representation.py:7]: awesome. maybe add that to the doc.

Comment (maxnoe) [reply fact/coordinates/representation.py:7]: done

**Review by kbruegge — DISMISSED** [review]

**Comment by dneise** [comment]
I cannot really comment on the inner workings, but the feature is according to the tests, fairly easy to use und certainly useful for a greater audience. So it should definitely go into pyfact.

**Comment by maxnoe** [comment]
Plotting a skymap of crab with the current python analysis tools:

```python
from fact.io import read_data
from fact.coordinates import camera_to_equatorial
from astropy.coordinates import SkyCoord
import astropy.units as u
import pandas as pd
import matplotlib.pyplot as plt

crab = SkyCoord.from_name('crab')

df = read_data(
    'build/crab_precuts.hdf5',
    key='events',
    columns=[
        'unix_time_utc',
        'zd_tracking', 'az_tracking',
        'reconstructed_source_position',
        'gamma_prediction'
    ]
).query('gamma_prediction >= 0.85').copy()

df['time'] = pd.to_datetime(df.unix_time_utc_0 * 1e6 + df.unix_time_utc_1, unit='us')

# new function of this PR
ra, dec = camera_to_equatorial(
    df.reconstructed_source_position_0,
    df.reconstructed_source_position_1,
    df.zd_tracking,
    df.az_tracking,
    df.time,
)

ra =  ra * u.hourangle
dec = dec * u.deg

plt.hist2d(
    ra.to('deg').value,
    dec.to('deg').value,
    bins=50,
    range=[
        [crab.ra.deg -1.5, crab.ra.deg + 1.5],
        [crab.dec.deg -1.5, crab.dec.deg + 1.5]
    ],
)

plt.xlabel('right ascencion / degree')
plt.ylabel('declination / degree')

plt.plot(crab.ra.deg, crab.dec.deg, 'o', markerfacecolor='none', ms=15, mew=1, color='C1')
plt.gca().set_aspect(1)
plt.colorbar()
plt.savefig('skymap.png', dpi=300)
```

Result:

skymap

**Review by dneise — APPROVED** [review]

**Pull request merged**